suggested_libraries.md

```diff
@@ -30,7 +30,23 @@ end
 
 ## :gem: Suggested NPM packages
 
+
+```
+yarn add rails-ujs turbolinks autoprefixer
 ```
-yarn add rails-ujs turbolinks
+
+To configure `rails-ujs` add the following to `application.js`:
+
+```js
+import Rails from 'rails-ujs';
+Rails.start();
+```
+
+To configure `turbolinks` add the following to `application.js`:
+
+```js
+import Turbolinks from 'turbolinks';
+Turbolinks.start();
 ```
 
+To configure `autoprefixer` add the following to `environment.js`:
```

Comment:
Example got lost here :)

Reply:
oh yep. I removed this part since we don't need autoprefixer.